src/objective.jl

```diff
@@ -115,12 +115,12 @@ However, it is not convertible to a variable.
 julia> JuMP.objective_function(model, JuMP.VariableRef)
 ERROR: InexactError: convert(MathOptInterface.SingleVariable, MathOptInterface.ScalarAffineFunction{Float64}(MathOptInterface.ScalarAffineTerm{Float64}[ScalarAffineTerm{Float64}(2.0, VariableIndex(1))], 1.0))
 Stacktrace:
- [1] convert at /home/blegat/.julia/dev/MathOptInterface/src/functions.jl:393 [inlined]
- [2] get(::JuMP.JuMPMOIModel{Float64}, ::MathOptInterface.ObjectiveFunction{MathOptInterface.SingleVariable}) at /home/blegat/.julia/dev/MathOptInterface/src/Utilities/model.jl:259
- [3] get at /home/blegat/.julia/dev/MathOptInterface/src/Utilities/universalfallback.jl:105 [inlined]
- [4] get at /home/blegat/.julia/dev/MathOptInterface/src/Utilities/cachingoptimizer.jl:436 [inlined]
+ [1] convert at /home/blegat/.julia/dev/MathOptInterface/src/functions.jl:398 [inlined]
```

Comment:
Surely this fails because we're not filtering out the path as well? Or does that happen somewhere else?

Reply:
We are filtering out this output so I was not required to update it.

Reply:
Oh, I missed the `.*` at the end of the regex. `r"^Stacktrace:.*"s` might be more clear?

Reply:
If I'm not mistaken, `.*` doesn't catch newlines

Reply:
The modifier `s` on the end makes it. I guess that's just as opaque... nm.